src/assetbundles/src/css/services.css

```diff
@@ -0,0 +1,80 @@
+.card-box {
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
@sharmarishabh1 can we change this to `align-items: flex-start;`

Reply:
Fixed here #530 

@sidedwards @bhupeshappfoster 
Please review the same whenever possible.